### PR TITLE
Write deserialisation tests for the api

### PR DESCRIPTION
**Pull request by lucasardonini**

### CODE REVIEW

File: mobile/src/main/java/ch/liip/timeforcoffee/TimeForCoffeeModule.java [hunk 1]
```diff
@@ -60,48 +60,28 @@ EventBus provideEventBus() {
     @Provides
     @Singleton
     TransportService provideTransportService() {
-
-        Gson gson = new GsonBuilder()
-                .setDateFormat("yyyy-MM-dd'T'HH:mm:ssZ")
-                .create();
-
+        Gson gson = new GsonBuilder().create();
```
Comment (FriedrichIII) [inline mobile/src/main/java/ch/liip/timeforcoffee/TimeForCoffeeModule.java]: @lucasardonini Are you sure about removing the dateFormat? Did you test it?

**Review by FriedrichIII — CHANGES_REQUESTED** [review]
Reviewed. See my comments.

File: mobile/src/main/java/ch/liip/timeforcoffee/TimeForCoffeeModule.java (hunk 1 above)
Comment (FriedrichIII) [reply mobile/src/main/java/ch/liip/timeforcoffee/TimeForCoffeeModule.java]: Tested and it actually breaks the timer countdown. Please fix.

**Comment by lucasardonini** [comment]
@FriedrichIII Applied your comment.

**Comment by lucasardonini** [comment]
@FriedrichIII Done changes

**Review by FriedrichIII — APPROVED** [review]

**Pull request merged**